src/ziggurat/streams.clj

```diff
@@ -209,7 +210,7 @@
 
     (when-not (nil? top)
       (KafkaStreams. ^Topology top
-                     ^Properties (properties stream-config)
+                     ^Properties (ssl-properties/build-ssl-properties (properties stream-config))
```

Comment:
we should add ssl configs in config.clj where we're adding properties for stream, consumer and producer APIs.